themes/default/content/blog/project-scoped-stacks-in-self-managed-backend/index.md

```diff
@@ -0,0 +1,187 @@
+---
+title: "Projects in Self-Managed State Backends"
+date: 2023-03-29
+meta_desc: Pulumi's Self-Managed backend now supports stack names scoped by project name.
+meta_image: meta.png
+authors:
+    - justin-vanpatten
+    - abhinav-gupta
+tags:
+    - features
+    - pulumi-releases
+---
+
+At Pulumi, we aim to offer our users the best-in-class IaaC experience.
+We recently shipped a series of impactful enhancements to our Service backend
+(check out [Pulumi Deployments](/blog/pulumi-deployments/) if you haven't already!),
+and we have more exciting features coming soon.
+
+While our Service backend remains the best place to store your Pulumi state,
```

Comment:
instead of "best", perhaps something more descriptive such as "most convenient, reliable, and secure place"